bundles/org.openhab.binding.haywardomnilogic/src/main/resources/OH-INF/thing/bow.xml

```diff
@@ -17,7 +17,14 @@
 		</channels>
 
 		<properties>
-			<property name="Vendor">Hayward</property>
+			<property name="vendor">Hayward</property>
+			<property name="type">Hayward</property>
+			<property name="sharedType">Hayward</property>
+			<property name="sharedPriority">Hayward</property>
+			<property name="sharedEquipmentSystemID">Hayward</property>
+			<property name="supportsSpillover">Hayward</property>
+			<property name="sizeInGallons">Hayward</property>
+
```

Comment:
These are initialized with "Hayward" and in chlorinator with an empty string. I think it'd make sense to initialize all with an empty value, at least those which are unknown at compile time.